src/components/Menu/index.tsx

```diff
@@ -1,81 +1,110 @@
 import React from "react";
 import Image from "next/image";
+import styled from "styled-components";
+
+import {
+  Navigation,
+  NavigationList,
+  NavigationElement,
+} from "../styles/navigation";
+import { LinkMenu } from "../styles/links";
+import { ButtonMenu } from "../styles/button";
+import { ContainerBig } from "../styles/container";
 import Linkedin from "./LinkedIN.svg";
 
+const ContainerMenu = styled(ContainerBig)`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  justify-content: space-between;
+  height: 4.2rem;
+  margin: 3rem auto;
+  align-content: center;
+  @media (max-width: ${({ theme }) => theme.breakpoints.xs}) {
+    justify-content: center;
+  }
+`;
+const SocialMediaElements = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding-top: 6px;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    display: none;
+  }
+`;
 export const Menu = () => {
   return (
-    <div className="container container--big display__row nav__main">
+    <ContainerMenu>
```

Comment:
Raczej na odwrót : MenuContainer, i MenuButton - > bo to masz kontener na menu, a nie menu na kontener